site/features.md

```diff
@@ -86,7 +86,7 @@ Using the Weave plugin enables you to take advantage of [Docker's network functi
 Also, Weave’s Docker Network plugin doesn't require an external cluster store and you can start and stop containers even 
 when there are network connectivity problems.
 
-See [Integrating Docker via the Network Plugin](/site/plugin.md) for more details.
+See [Integrating Docker via the Network Plugin (Legacy)](/site/plugin.md) for more details.
```

Comment:
I think we should say in this section that there are two choices of plugin; keep it brief but have links to the detail on both.